app/helpers/releases_helper.rb

```diff
@@ -8,6 +22,14 @@ def release_label(project, release)
     )
   end
 
+  def status_glyphicon(status_state)
```

Comment:
prefix method+constants with `github_` or `commit_` since we have deploy status which is different

Reply:
also just `icon` since it does not matter what library we use and the other helper is called `icon_tag`

Reply:
^